### PR TITLE
Style design

### PR DESCRIPTION
**Pull request by espimadrazo**
I made the first additions on the design.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,231 +1,231 @@
...
-      Vestibulum sed quam mi. Donec ipsum tellus, lobortis semper.
-    </p>
-  </article>
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">  
+    <meta name="viewport" content="width=device-width, initial scale=1">
+    <link rel ="stylesheet" href="style.css">
```
Comment (mauriciopasquier) [inline index.html]: Don't leave spaces before `=`

File: style.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  font-family: inherit;
+}
+
+a {
+  text-decoration: none; 
+  color: inherit;
+}
+
+footer {
+  display:flex;
```
Comment (mauriciopasquier) [inline style.css]: Leave a space after `:` :P

**Review by mauriciopasquier — CHANGES_REQUESTED** [review]
This looks really nice!

In general, I would try to take advantage of [semantic tags](https://www.w3.org/QA/Tips/goodclassnames) in the html when possible. What I mean is that, for example, instead of adding a `"ul-index"` class to the `ul` element, just say it's an `"index"` and use `ul.index` as the CSS selector. Same for `"anchors-index"` (this should be singular by the way, you're saying "this element is an anchor"). Or better yet, say the `menu` is an `index` and use `menu.index ul a` as a selector if you want to style the `a`. Alternatively, you could use just class names if you want to be independent of the specific elements, but using the element name inside the class name is the worst of both worlds.

I also left a couple of syntax comments.

File: style.css (hunk 2 above)
Comment (espimadrazo) [reply style.css]: There, I fixed it.

**Review by mauriciopasquier — APPROVED** [review]

**Pull request merged**